php/libraries/NDB_BVL_Instrument.class.inc

```diff
@@ -116,18 +100,20 @@ abstract class NDB_BVL_Instrument extends NDB_Page
      * all other dates in the table that appear on other pages (if they are
      * listed in the array of date fields) - note by @dariovins
      */
-    var $dateTimeFields = [];
+    public $dateTimeFields = [];
 
     /**
      * Array of date fields that contain only month year fields in frontend with
      * a static value for the date
      */
-    var $monthYearFields = [];
+    public $monthYearFields = [];
     /**
      * Array of column names to be ignored by the double data entry conflict
      * detector.
+     *
+     * @access public
      */
-    var $_doubleDataEntryDiffIgnoreColumns
+    public $doubleDataEntryDiffIgnoreColumns
```

Comment:
this is also changing the variable name which is going to break any subclass or script that uses it.

Reply:
PHPCS has a rule where only privately scoped variable can be prefixed with an underscore. We can change this to private or disable phpcs. Which do you want?